components/Chips/src/TypographyThemer/MDCChipViewTypographyThemer.m

```diff
@@ -22,7 +22,8 @@ @implementation MDCChipViewTypographyThemer
 + (void)applyTypographyScheme:(nonnull id<MDCTypographyScheming>)typographyScheme
                    toChipView:(nonnull MDCChipView *)chipView {
   UIFont *titleFont = typographyScheme.body2;
-  if (typographyScheme.mdc_adjustsFontForContentSizeCategory) {
+  if ([typographyScheme respondsToSelector:@selector(useCurrentContentSizeCategoryWhenApplied)] &&
+      typographyScheme.useCurrentContentSizeCategoryWhenApplied) {
```

Comment:
We'll likely want to fall back to checking `typographyScheme.mdc_adjustsFontForContentSizeCategory` for now if `useCurrentContentSizeCategoryWhenApplied` isn't implemented.

Reply:
This themer didn't have support for mdc_adjustsFontForContentSizeCategory for previous releasees, so it will be a new behavior. Adding fallback for `mdc_adjustsFontForContentSizeCategory` would help people who are already using mdc_adjustsFontForContentSizeCategory and expecting it works for Chip theming extension in next release. Should we add it?

Reply:
Ah, true. Hm. Until we delete `mdc_adjustsFontForContentSizeCategory` I think we should consistently support it across our components.

What do you think?

Reply:
Agreed. I think adding consistent support across our components is the right move here.

Reply:
Addressed in 3c854303eae0ad54a66e5da505f32cd22c0b6c95